_posts/2020-06-17-ombulabs-open-source-guidelines.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "OmbuLabs Open Source Guidelines"
+date: 2020-06-17 11:16:00
+categories: ["open-source", "guideline"]
+author: bronzdoc
+---
+
+![goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg](https://tettra-production.s3.us-west-2.amazonaws.com/0d6efb4f154041e899af17bdcd19c1b5/ce8d5f452da02a88554d80c256162a35/d822b155a4112474fdb7aea5ee22465e/d28647d6b4d6e5842508679707ee21a7/goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg)
+
+At OmbuLabs we dedicate some of our work hours to open source every week. Also some of us contribute to open source during our spare time.
+
+Here are a few tips to that help us keep OSS contributions organized
+
+<!--more-->
+
+From now on every-time we contribute to an OSS project in working hours, we fork it to the [FastRuby.io organization](https://github.com/fastruby).
+
+This way we can keep track of the contributions we made as a company, also the branches created in the project are accessible to the rest of the team.
+
+As a company we get these benefits:
+
+\- We use OSS projects and we want them to be better
+
+\- We want our brand to be associated with these OSS projects
+
+\- We want our teammates to learn more about these projects
+
+\- We want our contributions to be noticed
+
+\- We want to use our contributions in the future
+
+\- Sometimes contributions take a long time to be merged into the main OSS project. This way we make sure that we include references to our company’s organization.
+
+\- None of our references will point to a person’s repository. They will point to one of our company’s fork.
+
+As an individual we get to keep the benefits too.
+
+\- Display the user in the PR page
+
+\- Display the contribution in the user's contribution graph
+
+\- Display commits authored by the original user
+
+# README
+
+When creating a new OSS project it is good to have a good README. The readme should include steps on how to install the project, if you want to contribute it should have steps on how to contribute to their projects and a code of conduct
```

Comment:
This sentence is not clear to me. It starts saying that someone is creating an OSS project, but then it says `if you want to contribute` (maybe it's `if you want contributors`?) and then it says `how to contribute to their projects` (but wasn't it this person's new created project?).